README.md

```diff
@@ -60,6 +60,14 @@ plugins=(
 
 Restart your terminal.
 
+### Fig 
+
+[Fig](https://fig.io) adds apps, shortcuts, and autocomplete to your existing terminal.
+
+Install `zsh-yarn-autocompletions` in just one click.
+
+<a href="https://fig.io/plugins/other/zsh-yarn-autocompletions_g-plane" target="_blank"><img src="https://fig.io/badges/install-with-fig.svg" /></a>
```

Comment:
Will it break if we use Markdown link here instead of HTML `<a>` tag?

Reply:
Yes, it would break. I used `<a>` since the link encompasses the install button `<img>` below.